lib/src/generated/bledata.pb.dart

```diff
@@ -3,7 +3,7 @@
 //  source: bledata.proto
 //
 // @dart = 2.12
-// ignore_for_file: annotate_overrides,camel_case_types,unnecessary_const,non_constant_identifier_names,library_prefixes,unused_import,unused_shown_name,return_of_invalid_type,unnecessary_this,prefer_final_fields
+// ignore_for_file: annotate_overrides,camel_case_types,unnecessary_const,non_constant_identifier_names,library_prefixes,unused_import,unused_shown_name,return_of_invalid_type,unnecessary_this,prefer_final_fields,prefer_constructors_over_static_methods,constant_identifier_names
```

Comment:
```dart
//  Generated code. Do not modify.
```

I prefer to respect this advice. What happens if we keep the file as is? Decrease in the health-rating on pub.dev? I'd rather consider accepting that, instead of manually modifying generated files (every time).

Please, strongly consider reverting all changes to the generated files.

Reply:
it has to do with the fact that linting fails on latest dart version. I already noticed that  @jonahwilliams  created an [issue](https://github.com/dart-lang/protobuf/issues/506). I also tend to wait until the issue is fixed in protobuf and integrate it then .